scripts/case_selection/workflow_case_selection.R

```diff
@@ -22,34 +22,34 @@ option_list <- list(
               help="BPC cohort"),
   make_option(c("-s", "--site"), type = "character",
               help="BPC site"),
-  make_option(c("-u", "--save_synapse"), action="store_true", default = FALSE, 
-              help="Save output to Synapse")
+  make_option(c("--production"), action="store_true", default = FALSE,
```

Comment:
Updated to production since I just made it such that we are always storing the files somewhere.

That said, the staging folder is unfortunately hardcoded currently.  The "right" way to do this would probably be to have a "staging_config.yaml"